LiveController_deployment.md

```diff
@@ -0,0 +1,91 @@
+## Machine prerequisites:
+====================================================
+liveController & liveRecorder can be installed in single or separate machines.
+
+## Software prerequisites:
+====================================================
+ - Centos 6.8+
```

Comment:
wrote CentOS 6.8+ so I'm changing to RHEL/CentOS 7 or above or Ubuntu 16.0.4 or above